docusaurus.config.js

```diff
@@ -222,11 +247,11 @@ module.exports = {
             "https://github.com/camunda/camunda-platform-docs/edit/main/",
           beforeDefaultRemarkPlugins: [versionedLinks],
           // 👋 When cutting a new version, remove the banner for maintained versions by adding an entry. Remove the entry to versions >18 months old.
-          versions: {
-            "8.0": {
-              banner: "none",
-            },
-          },
+          // versions: {
+          //   "8.0": {
+          //     banner: "none",
+          //   },
+          // },
```

Comment:
commented out because I removed all versions except the most recent ones (8.1 / 3.9.0)